orchestrator/orchestrator/src/main_loop.rs

```diff
@@ -178,6 +181,15 @@ pub async fn eth_oracle_main_loop(
             }
         }
 
+        let latest_concise_block = web3
+            .eth_get_block_by_number(latest_eth_block.unwrap())
+            .await;
+        if let Ok(block) = latest_concise_block {
+            if block.total_difficulty >= merge_terminal_total_difficulty {
+                return;
```

Comment:
we should print a log message stating that we've detected eth2 difficulty and the oracle has been paused